inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/DefaultMessageSender.java

```diff
@@ -191,6 +195,10 @@ public String getSDKVersion() {
         return ConfigConstants.PROXY_SDK_VERSION;
     }
 
+    public void enableDataTruncation(boolean enableDataTruncation) {
+        this.enableDataTruncation = enableDataTruncation;
```

Comment:
Too large data has 3 solution.
1. Truncation. If the data has been encoded or compressed, truncation data is wrong.
2. Reject the request. It is better.
3. Sort process large data. DataProxy will has some risk.

Reply:
So should we replace the truncation policy with the reject policy？

It seems that it is not easy to tell whether the data can be truncated at the sending end.
I think if the receiver can't handle the truncated content, it should also return an error message to the sender